Makefile

```diff
@@ -19,12 +24,12 @@ install-base:
 .PHONY: install-base
 
 
-all-dom0:
-	$(MAKE) all -C daemon
+all-dom0: all-base
+	+$(MAKE) all -C daemon
 .PHONY: all-dom0
 
-install-dom0:
-	$(MAKE) install -C daemon
+install-dom0: install-base all-dom0
```

Comment:
Adding `install-base` breaks the package build since it installed stuff which is not part of the rpm. Try building with an up to date qubes-builder.